pkgs/applications/virtualization/vmware-workstation/default.nix

```diff
@@ -49,8 +49,8 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "vmware-workstation";
-  version = "16.2.3";
-  build = "19376536";
+  version = "16.2.4";
```

Comment:
What about an update to 17.0.0?

Reply:
I agree, maybe we should just backport 17.0.0 update from master, there is no breaking changes in that _major_ release anyway.